.github/workflows/run_cucumber_tests.yml

```diff
@@ -26,10 +26,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Ruby 2.5.7
+    - name: Set up Ruby 2.5.8
       uses: actions/setup-ruby@v1
       with:
-        ruby-version: 2.5.7
+        ruby-version: 2.5.8
```

Comment:
So it looks like specifying the patch version doesn't do anything/is ignored when using this action, so maybe this should just be '2.5' or we use a different action that allows specifying the patch version